src/lib/config.js

```diff
@@ -1,6 +1,6 @@
 import _ from 'lodash';
 import path from 'path';
-import envConfig from './environment';
+import envConfig from '@lib/environment'
```

Comment:
Should we be using this pattern in other modules?

Reply:
we can, I added this 'absolute' import to the webpack config mainly to avoid having to use long `../../../` relative imports in the testing package. Short relative imports are fine though I think.